packages/js/index.ts

```diff
@@ -3,7 +3,7 @@ import Verto from './src/Verto'
 import { setAgentName } from '../common/src/messages/blade/Connect'
 import CantinaAuth from '../common/src/webrtc/CantinaAuth'
 
-export const VERSION = '1.2.9'
+export const VERSION = '1.3.0'
```

Comment:
Yes, as we discuss I believe `1.2.10` will be  safer. Although we introduced a new public param.

Reply:
my 2 cents for 1.3.0: we are following [semver](https://semver.org/) which suggests
 - MINOR version when you add functionality in a backward compatible manner
 - PATCH version when you make backward compatible bug fixes

Reply:
(mostly useful when you scan the changelogs)

Reply:
@danieleds The confusion it's because there are existing 1.3.0.alpha.X and 1.3.0.cantina.X versions already published.

IMO since we add functionality, but there is published 1.3.0.XX maybe we just publish a 1.3.1 with the new change.

I also have confirmed that  1.3.0.cantina.62 is from the old SDK.

Reply:
got it, thanks @jpsantosbh !